src/content/docs/fr/reference/errors/middleware-cant-be-loaded.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: Impossible de charger le middleware.
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+import DontEditWarning from '~/components/DontEditWarning.astro'
+
+
+> **MiddlewareCantBeLoaded**: Le middleware a généré une erreur pendant qu'Astro essayait de le charger.
```

Comment:
I am not sure, as I have never developed in French, let alone another language before. When the error is thrown, I - however naively - assume it isn't translated.

Should this part be translated, @sarah11918? Or would it be more helpful to keep it this way?

Reply:
Hello @VoxelMC ! This error code must not be translated yep!

https://github.com/withastro/docs/blob/main/src/content/docs/en/reference/errors/middleware-cant-be-loaded.mdx